ext/ffi/ffi.stub.php

```diff
@@ -33,10 +33,16 @@ public static function arrayType(FFI\CType $type, array $dimensions): ?FFI\CType
     /** @prefer-ref $ptr */
     public static function addr(FFI\CData $ptr): FFI\CData {}
 
-    /** @prefer-ref $ptr */
-    public static function sizeof(object $ptr): ?int {}
+    /**
+     * @param FFI\CData|FFI\CType $ptr
+     * @prefer-ref $ptr
```

Comment:
Why do we `prefer-ref` here`? Couldn't this be normal by-val instead? /cc @nikic, @dstogov

Reply:
It's bad API design, but unfortunately the prefer-ref is needed :( It causes the argument to be fetched for write, and FFI\CData will treat such fetches differently (by returning another FFI\CData object, rather than a plain value).

Reply:
Ah, I see. Thanks for the explanation!